src/main/resources/label-definitions.properties

```
@@ -458,6 +458,7 @@ ibm-security-appscanstandard-scanner=post-build misc
 icescrum=report external
 icn-plugin-loader=misc
 icon-shim=ui
+icq-notifier=notifier
```

Is this the correct plugin name? I would expect…

```suggestion
icq-notification=notifier
```